package.json

```diff
@@ -12,6 +12,7 @@
     "lodash": "^4.17.11",
     "node-rsa": "^1.0.5",
     "request": "^2.72.0",
+    "axios": "^0.19.2",
```

Comment:
I think I have PTSD from this library.

Look no further than:  https://github.com/axios/axios/blob/master/lib/core/mergeConfig.js#L13-L72

I don't know what to recommend instead, but I would strongly recommend to avoid axios.

Reply:
node's own http library is pretty trivial to get working, I use it whenever possible to connect to API endpoints